docsteady/templates/dm-tpr.latex.jinja2

```diff
@@ -142,7 +142,12 @@ The following personnel are involved in this test activity:
   \end{itemize}
 {% endfor %}
 \end{itemize}
-\item Additional Test Personnel involved: None
+\item Additional Test Personnel involved:
+  \begin{itemize}
+  {% for testcase in testcases_map %}
```

Comment:
These lines can also be written like this:

```
  {% for testcase_key, testcase in testcases_map.iteritems() %}
      \item Test case \href{https://jira.lsstcorp.org/secure/Tests.jspa#/testCase/{{ testcase_key }}{{ '}' }}{{ '{' }}{{ testcase_key }}{{ '}' }}: {{ testcase.test_personnel }}
  {% endfor %}
  \end{itemize}
```

...but it's fine the way it is mostly, although I'd change:
`testcases_map[testcase]['custom_fields']['Test Personnel']` to `testcases_map[testcase].test_personnel`

Reply:
Thank you Brian for your review.

The first implementation you suggested gives the following error.
  File "<template>", line 147, in top-level template code
jinja2.exceptions.UndefinedError: 'dict object' has no attribute 'iteritems'

I have implemented the second one, and it seems to work fine.
I am going to merge this branch to master.